test/testrunner.py

```diff
@@ -158,15 +160,14 @@ def main():
         if specific_test_type == "sanity":
             pull_dlc_images(all_image_list)
         if specific_test_type == "eks":
-            eks_terminable_clusters = setup_eks_clusters(dlc_images)
+            new_eks_cluster_name = setup_eks_clusters(dlc_images)
```

Comment:
nit: let's name this something like "eks_test_cluster" or "eks_cluster_name" -- the "new" is not needed